index.html

```diff
@@ -402,33 +407,33 @@ <h3 class="supporting-text">Let's connect</h3>
         </article>
       </section>
 
-      <footer class="contact-wrapper" id="contact">
-        <div class="contact-card">
-          <h2 class="title">Contact me</h2>
-          <p class="contact-note">
-            If you have an application you are interested in developing, a
-            feature that you need built or a project that needs coding. I’d love
-            to help with it
-          </p>
-          <form action="https://formspree.io/f/xknylwwy" method="POST">
-            <input
-              type="text"
-              name="name"
-              placeholder="Name"
-              maxlength="30"
-              required
-            />
-            <input type="email" name="email" placeholder="Email" required />
-            <textarea
-              name="message"
-              placeholder="Write your message here"
-              maxlength="500"
-              required
-            ></textarea>
-            <button type="submit" class="primary-btn">get in touch</button>
-          </form>
-        </div>
-      </footer>
     </main>
+    <footer class="contact-wrapper" id="contact">
+      <div class="contact-card">
+        <h2 class="title">Contact me</h2>
+        <p class="contact-note">
+          If you have an application you are interested in developing, a
+          feature that you need built or a project that needs coding. I’d love
+          to help with it
+        </p>
+        <form action="https://formspree.io/f/xknylwwy" method="POST">
+          <input
```

Comment:
- [ ] [OPTIONAL] it would be useful to add a min length Validation to the name to limit single character values to be submitted instead of the name.

Reply:
That's a great idea, except you Kanye West, since his name might change to Y, from Ye, LOL 😄 .
Thank you for your feedback @isaacpitwa